rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -836,7 +836,7 @@ def _post_process(self, refresh_balances: bool) -> None:
                 message_type=WSMessageType.REFRESH_BALANCES,
                 data={
                     'type': 'blockchain_balances',
-                    'blockchain': self.evm_inquirer.chain_id.to_blockchain().serialize(),
+                    'blockchain': self.evm_inquirer.chain_id.to_blockchain().value,
```

Comment:
As we discussed in discord we use  uppercaase when sending the blockchain

Reply:
Using `.value` is a terrible approach. It's a an enum and its serialization should be used. Not adding hacky code like this. As I asked please adjust for the incosistencies across both endpoints. Something is wrong if one endpoint is giving the supported types to the frontend in one way and the other in another

Reply:
We use .value in many places but which if these have to do with serialisation towards the API and the outside?

As I said fix consistency here. This is a problem